### ISSUE TITLE
No option to bring add image by inserting a URL

### ISSUE THREAD

**Issue by knitgrandeur** [elementor/elementor#413]
hello,
I love your plugin but can't use it to create posts because the fastest way to create them is by inserting an image URL. Could you pls add this capability to your plugin.
thanks

**Comment by yehudah** [comment]
Sorry don't understand the request.

**Comment by yehudah** [comment]
I think I understand, and we don't have this option, maybe in the future.

**Comment by ZalmanBlau** [comment]
+1

**Comment by gioglifx** [comment]
I also agree with this feature to be added.  Access Images from ext. URL in EVERY module with images.  How can this be implemented if this is still not available yet?

**Comment by blue928** [comment]
+1 for this

We purchased Pro and are experimenting with Elementor as a replacement for Divi. We love Elementor so far, but to be quite honest, this is one of the features Divi has that prevents us from making the jump. Such a small thing is such a major improvement in efficiency. This exact features is built into Divi in one or two places on all of the image modules. One can either paste a url right then and there into the module; or one can access the media area where "Insert from URL" is one of the items to select.

When building a Divi layout, we reference images we upload to a central server; As well, when we offer to redo client sites we reference their already hosted images from their current site.

Using Elementor hit us pretty hard in reminding us just how much more time it takes to to do the same layout when having to use browser Inspect, find the image link, download it to the computer, optimize it, upload it to the server. This workflow almost doubles the time an experience team member can do the same layout in elementor.

Adding an image url would let us make the jump not only for our all projects, but increase licenses purchases for Elementor Pro in that they'd need them for the WooCommerce functionality.

And while that would be an amazing improvement, I would say incorporating an option to actually save that to the local media library would take that even further. (Divi doesn't have that option)
Here are screenshots of the default built in Divi technology.

![screen shot 2018-04-14 at 11 07 49 am](https://user-images.githubusercontent.com/6431043/38769646-5791148c-3fd4-11e8-8a05-ec8d07a94406.png)
![screen shot 2018-04-14 at 11 08 11 am](https://user-images.githubusercontent.com/6431043/38769647-57a2887a-3fd4-11e8-84db-e804f82fc4c7.png)
![screen shot 2018-04-14 at 11 08 26 am](https://user-images.githubusercontent.com/6431043/38769648-57b348b8-3fd4-11e8-90e7-e19878c06e31.png)

**Comment by blue928** [comment]
Also to note, issue #1411 references accomplishing this by using an HTML element and inserting the source. But that's not practical at all. As well, it's impossible to apply that approach to background elements / images over which text will be laid.

**Comment by KRline** [comment]
I just stopped using Elementor and did not update my pro licence as this is a feature I really miss. Let's hope it does not take another year to implement!!! this problem has been trailing (being neglected) way too long!

**Comment by KimchaC** [comment]
I completely agree with this. It is really frustrating not to be able to do this.

Sometimes I want to use an image that is included with my theme for example without duplicating it. Duplicating it makes it uncachable...

**Comment by Hanleymade** [comment]
It seems like this would be an easy option to add, but one that many people want to use.

**Comment by lordloadalot** [comment]
Hello Again! is there any hope to get this feature under every „add image“ button now? My client wants to use exif.co - we just can offer him the html-module. NO Galleries NO Sliders NO backrounds ( row or section ) ... :( pleeease! We switched from DIVI one year ago. 13 installs right now. Transforming old sites from DIVI to Elementor at our own costs just to have a better workflowbut we need this feature. Use of CDNs is a common feature in 2018.
Best Martin

**Comment by Hanleymade** [comment]
So the original request is now TWO years old. Why is this still not being added to Elementor??

**Comment by lordloadalot** [comment]
JUST DREAMIN´  ... :)

![uploadfromurlelementor](https://user-images.githubusercontent.com/29510055/42687948-c5e042ac-869a-11e8-8a3f-18abf27558b1.jpg)

**Comment by Hanleymade** [comment]
@lordloadalot Exactly - given this has been a plugin option in WP for years, why not add it to Elementor??

**Comment by gioglifx** [comment]
Hopefully.  This is much needed.  SEO and workflow demand this...   👍   :)

**Comment by MiMoHo** [comment]
Pushing this issue. Hard to believe this has not been implemented yet! Inserting an external image is a must-have!

**Comment by dennis1088** [comment]
+1 on this feature would like to use CDN images to decrease the load on our wordpress servers

**Comment by RuebenTT** [comment]
My issue is WORSE - Whenever I open the library from a module the library is constantly loading and has no thumbnails

**Comment by fattusmaximus** [comment]
Bump. Please add.

This is especially useful for multisite so I'm not wasting server space for the same images.

Using the WP image module with CSS does the trick, but it's awful for the workflow.

**Comment by zweiklang** [comment]
Bumped.

Guys, this is absolutely overdue. We'd also need this!

**Comment by lordloadalot** [comment]
My favourite thread! Feelin' quite familiar here. AFTER ONE YEAR :) I know it. One day ...

**Comment by gioglifx** [comment]
Are any contributors or anyone from Elementor even reading these?  Are these simply ignored?  At the very least acknowledge if yes or no.  At this point, the word "no" would be disheartening but, at least we'd know if it'll be implemented or not.  Any thoughts?

**Comment by lordloadalot** [comment]
It is coincidence. We are just the guys at the table who didn't get their Pizza. I must admit I [love](https://medium.com/@lordloadalot/thank-you-elementor-d6214c3e8613) the product and I will not change my toolkit again. No matter if URL-embed wil come or not ;)

**Comment by godvsgodvs** [comment]
This lack of response from the developer is the main reason I cannot make a permanent switch from Divi to Elementor. They seem to be totally tone-deaf on certain issues. On the other hand, Elegant Themes has been that way on custom headers, footers and dynamic content for at least 2 years.
As Casey Stengel said, "Can't anybody here play this game?".

**Comment by godvsgodvs** [comment]
In my frustration I found this plugin. So far, so good. And it seems to work anywhere.
https://wordpress.org/plugins/external-media-without-import/

**Comment by lordloadalot** [comment]
There is another reason for URLs. Simply to check wich image is embedded when working with visually similar files. ( At the end of URL: for the version or format of the file - at the beginning of URL: to check https:// ). You can just choose a new image, that's it.

**Comment by godvsgodvs** [comment]
I guess the bottom line is there are no "perfect" page builders/theme builders/site builders out there. It will always probably be picking the lesser of several evils.
Regarding the 1M installs, that of course refers to the free version. I would hope the pro version requests would have a higher priority.

**Comment by gioglifx** [comment]
@lordloadalot - the reason for such a plugin is for a faster flow, rather than "dreaming of an overnight webdesigner carreer".  Yes, there may be some that [need] a plugin rather than be able to develop.  But, with that in mind - Sir, you would fit that category for the mere fact that you're here.  Unfortunately, negative comments that don't enhance the thread do nothing but bloat the thread with no progress.  I have the PRO version, and that doesn't mean that having a plugin makes my websites less developed.  A general contractor does not do the work himself - though he knows how to do it, but subcontracts to speed things up.  
Or... it's like assembling a car, for example:  unless you make the engine from the ground up, machine ALL the parts, make the glass, etc - you use other pre-made things to speed things up or for many other reasons that do not involve dreaming.  
This thread is very similar, in regards to speed things up for the developing and designing a website without much hand coding.  For that Sir, shying away from Wordpress might be a better option and hand-code everything.  As far as the thread, the people here (I'm strongly assuming) are asking for a very standard option to use a CDN for SEO purposes, and speed of course.   By the way, was that a solution offered?  I wasn't sure if the URL comment was - (if it was, would you mind elaborating please) - Hope that helps.

**Comment by lordloadalot** [comment]
:) @gioglifx you missed it. https://medium.com/@lordloadalot/thank-you-elementor-d6214c3e8613.
besides: https://github.com/pojome/elementor/issues/4065.
And last: "... do nothing but bloat the thread with no progress"  hahaa :D
![bildschirmfoto 2018-09-25 um 09 51 37](https://user-images.githubusercontent.com/29510055/46000359-9e705200-c0a8-11e8-97d4-c2d5e183bb3b.png)

**Comment by awithy** [comment]
I completely agree this is a must have feature.  My company just built a new website with Elementor and have just now realized this issue.  This will probably be a complete show-stopper for us, as we must have the ability to host images from our CDN.  Please prioritize this very important feature.  It will be impossible for me to recommend Elementor other than to mom-and-pop local businesses because of the lack of this feature.

**Comment by awithy** [comment]
> In my frustration I found this plugin. So far, so good. And it seems to work anywhere.
> https://wordpress.org/plugins/external-media-without-import/

Thank you!!!  I was about to throw away Elementor before I found this plug-in.  This seems to be such an important requirement in order to use CDNs.

**Comment by portanicolas** [comment]
+1 I also need the same functionality

**Comment by portanicolas** [comment]
> Bump. Please add.
> 
> This is especially useful for multisite so I'm not wasting server space for the same images.
> 
> Using the WP image module with CSS does the trick, but it's awful for the workflow.

it's actually the best way to get around the problem

**Comment by 66logic** [comment]
+1 and... still waiting on this.

**Comment by ducfather** [comment]
I am using this on a multisite setup and without this option the server space would explode soon...
If this feature should be implemented somewhen in the future, could you add it to the media carousel too please? :)

**Comment by coollorenzo** [comment]
Would be very usefull!!

**Comment by inred-froehlich** [comment]
Yes plaes, i need this feature. And please make the URL Input Field Shortcode Ready. So i could insert a URL Link from a Shortcode from a Meta Field. Thank you very

**Comment by evanomondi** [comment]
Useful feature.

**Comment by wcmenk01** [comment]
Agreed with the above.  Having to upload all images to the site is time-consuming and inefficient

**Comment by taropaa** [comment]
+1 This feature is needed urgently! It's like having a nice Lambo but it's missing the windshield.

**Comment by pabloaugusto** [comment]
+1 here... there is no sense a builder without this!

**Comment by faynburd** [comment]
+1 
Very useful feature - hard to use the Elementor without it.
Considering to move to Thrive Architect

**Comment by samheavyside** [comment]
How has this issue been open for nearly _3 years_ and not been resolved!? That is insane! In the meantime, you've developed an entire management system so we can use Dynamic content to populate the image source (including integrations with ACF and other custom field managers), yet you haven't given us the ability to simply _type in a URL_? Come on...

**Comment by pabloaugusto** [comment]
> How has this issue been open for nearly _3 years_ and not been resolved!? That is insane! In the meantime, you've developed an entire management system so we can use Dynamic content to populate the image source (including integrations with ACF and other custom field managers), yet you haven't given us the ability to simply _type in a URL_? Come on...

Yes! Its ridiculous!
I need to implement svg sprites with http://site.com/sprite.svg#imageID and its been a nightmare becouse i cant put a simple image link... =/

**Comment by pingram3541** [comment]
+1

also would like the ability to use relative urls too please

**Comment by pjeby** [comment]
Here's a (very experimental) hack you can use to do this.  It checks to see if the string entered in the media search box is an absolute URL with a host, path, and scheme.  If so, it returns a fake image object for that URL, that the media control (used for images, backgrounds, etc.) will then accept and save.

I've only just scribbled and tried this myself today, so I don't know if there are any significant bugs or downsides, except that **sometimes you have to clear the search box and re-paste the URL for the image to show up**.  (I don't know why that is yet.)  But it's already a huge improvement in workflow for me when working with Unsplash to prototype something; it's even better than the "External Media Without Import" plugin, since it doesn't require a separate pseudo-uploading step.

In order to get this to work, it has to be added to a file that's executed during admin-ajax requests; I have it in a plugin (auto-created by [Imposer](/dirtsimple/imposer/)), but you *might* be able to get it to work in `functions.php`.

```php
if ( wp_doing_ajax() ) add_action('wp_ajax_query-attachments', function() {
    if ( ! current_user_can( 'upload_files' ) ) wp_send_json_error();
    $query = isset( $_REQUEST['query'] ) ? (array) $_REQUEST['query'] : array();
    if ( ! array_key_exists('s', $query) ) return;
    $required = array_flip(['scheme', 'host', 'path']);
    $url = $query['s'];
    if ( count( array_intersect_key($required, parse_url($url)) ) < 3 ) return;
    if ( empty( $size = @getimagesize($url) ) ) return;
    list( $type, $subtype ) = explode( '/', $size['mime'] );
    $data = [
        'id' => $url,
        'url' => $url,
        'mime' => $size['mime'],
        'type' => $type,
        'subtype' => $subtype,
        'editLink' => false,
        'meta' => false,
        'height' => $h = $size[1],
        'width'  => $w = $size[0],
        'orientation' => $o = $h > $w ? 'portrait' : 'landscape',
        'sizes' => [
            'full' => [ 'url' => $url, 'height' => $h, 'width'  => $w, 'orientation' => $o ]
        ]
    ];
    wp_send_json_success( [$data] );
}, 0);
```

This code doesn't support relative URLs (yet), but I hope to add that soon.  I'm thinking that if you enter a URL like `//myhostname.com/whatever.png`, and `myhostname.com` is the current host, then the URL passed to elementor would be `/whatever.png`.  Or maybe if you do something like `relative://_/whatever.png`.  (Whatever the case, the code would have to use an absolute URL for the `getimagesize()`, but put a relative URL in the `id` and `url` fields.)

**Comment by erizone** [comment]
Very much needed when using images hosted outside of your websites server!

**Comment by N4rsha** [comment]
+111111111111111111111111111111111

**Comment by RBucci** [comment]
Elementor is suggesting we start using their plugin without the CDN option, without external image linking, without the possibility of saving space on our servers. I'm going ahead towards WPBakery. What a letdown, guys. Over 3 years has passed and you couldn't add a single line of effing code to please a hell of a lot of users because you're either too lazy to do it or because you want to sell an extra option that enhances THIS BASIC functionality. KUDOS! -1

**Comment by lmurr248** [comment]
So I managed to set it up so that when I send an image URL via an API to my site it dynamically appears as an image in the post using elementor. I did this by adding a shortcode to convert an image url to an image. This would work on any image url.

The link to where I got the code for this shortcode is here: https://stackoverflow.com/questions/42479022/wordpress-simple-image-shortcode.

I then set up the custom field for the URL as a plain text field using Pods, and then added the shortcode element in elementor and dynamically linked to it within the shortcode element and used the Before and After fields to complete the shortcode itself.

The code is below, and the actual shortcode added into elementor is:
`[img src="http://example.com/wp-content/uploads/2017/02/hello.jpg"]`

`// Add Shortcode
function img_shortcode($atts)
{
    // Attributes
    $atts = shortcode_atts(
        [
        'src' => '',
        'link_to_img' => '',
        ], $atts, 'img'
    );

    $return = '';
    if ($atts['link_to_img'] == 'yes')
    {
        $return = '<a href="' . $atts['src'] . '">
                    <img src="' . $atts['src'] . '"/>
                </a>';
    }
    else{
        $return = '<img src="' . $atts['src'] . '"/>';
    }
    // Return HTML code
    return $return;
}

add_shortcode('img', 'img_shortcode');`

**Comment by yorkfrog** [comment]
+1 - just started using image CDN.  It picks up images, but not backgrounds and suddenly it's "how do I do that then?".  So, yes please, image processing to handle all the devices, sizes, compression, etc is getting so complex being able to hand it off to an external service is a massive relief.

**Comment by windyOC** [comment]
+1 ,I also wanted using image CDN,but I can't find anyway to pick up my image by url

**Comment by KimchaC** [comment]
If you guys want to use a CDN this is not the correct way of doing it. What we really need is a filter that lets us make changes to the CSS before it gets saved.

That would be very easy to implement for the Elementor team (much easier than this).

Beaver Builder for example supports this.

I have opened a new ticket for this here. Perhaps you guys can comment on it so it gets some traction:
https://github.com/elementor/elementor/issues/10126

**Comment by secondsky** [comment]
I am using WP Offload Media. Works great with Elementor besides one "Bug". I noticed that if Lightbox functionality of Elementor is used, that the URIs are not rewritten for these images. I guess its because the URIs are somehow put directly into Javascript when the page is rendered.
Elementor should add some functionality to add a CDN path or make other appropriate changes so that other Plugins like WP Offload Media can rewrite the URIs to the CDN/S3 Paths

Changing URIs hard is not a good idea btw... rewriting is a way better solution.

So hopefully the Elementor Team finally does something about this issue, its 2020, CDN is a musst!

**Comment by Crecera** [comment]
Please add this already!  Over 3 years now is ridiculous and it has tons of obvious use cases.  Here's another:  I shifted to Elementor from Divi, because of how it more seamlessly works with dynamic content through ACF, but I really need to be able to dynamically update images from Google Sheets using WP All Import for image fields and galleries.  Please listen to your advanced users and add this feature.

**Comment by spedepekka** [comment]
+1

**Comment by Morazajuan** [comment]
+1 man is that really hard?

**Comment by gerardreches** [comment]
+1!!!!

**Comment by sonicboom81** [comment]
This thread was started in 2016, and no option yet to use an external URL as an image. Please implement it. In my case I want to use an ACF URL Field to show an image in Elementor without having to use code.

**Comment by sonicboom81** [comment]
After testing a few plugins with Elementor, there's two that allow for you to insert an image using just an URL when the URL is in an Advanced Custom Field (there's other options too). Both paid plugins - Jet Engine and Anywhere Elementor Pro. These plugins deal with dynamic content, which Elementor Pro doesn't do as well. It's a shame Elementor Pro requires a paid plugin to achieve this.

**Comment by Crecera** [comment]
@sonicboom81 .  Thanks for looking in to this.  I'm trying to do something similar to you with ACF.  I have Anywhere Elementor Pro and will look at using for this, but to clarify, I believe this will only work in the Anywhere Elementor widgets and not globally, right?   It doesn't look like Anywhere Elementor supports image galleries or carousels, so I may need to look at Jet Engine.

**Comment by ceIia** [comment]
this has been going on since 2016... how hard could it be? or at least create a way of using S3/Cloudfront images -.-

**Comment by domihagen** [comment]
+1 we want to use Cloudinary with Elementor...

**Comment by KRline** [comment]
+1  
For now, I just add plain <HTML> to display images in elementor. No need even to have them in WP media via any plugin. Advantage is that I can add a link to the full image on cloudinary for fancybox while only using the downsized image for fast download in web page. Exceptions: (logo and) backround images need to be in WP media.
Example below: <a> points to the full size image,   <img> points to the downsized image with a width of 450 pixels (c_scale,w_450) resizing done on the fly by cloudinary.

<a href="https://res.cloudinary.com/account/image/upload/v1234567890/image.jpg" >
<img class="aligncenter" src="https://res.cloudinary.com/account/image/upload/c_scale,w_450/v1234567890/image.jpg" alt="image alt text" title="image title" width="450" /></a>

Another option option would be for Elementor to make it possible to edit plain HTML (with classic editor) without messing it up the whole lay-out. This would also be great to edit small typing errors quickly without having to load the whole Elementor overhead.

**Comment by zevrix** [comment]
just add my voice to request this feature. thanks.

**Comment by saberlight** [comment]
+1 Although, I think this is actually better for an integrated core WordPress function, but definitely needed nonetheless.

**Comment by sonicboom81** [comment]
@Crecera I have tried Elementor Anywhere Pro before, and I think it may do what you want, but as EA Pro does not work with Crocoblock's JetSmartFilters (whereas Jet Engine does) I had to stop using EA Pro. I think you are right in saying it will only work within the widget - it doesn't change Elementor Pro's functionality. Neither does Jet Engine. I plan to use Jet Engine for my website for most things, as it can do a lot of stuff, and means I can get rid of a few other plugins and improve speed (including some other Jet plugins like JetBlog), but like many plugins it still has a number of bugs.

**Comment by slaidel** [comment]
Such a thing could have been solved there in 2016 ... we are already in 2020.

Please folks pay attention to this request. **simple** but **important**

**Comment by spedepekka** [comment]
I don't know if I should laugh or cry at this point xD

**Comment by slaidel** [comment]
To solve this problem I used the [External Media without Import] plugin, since Elementor hasn't solved it yet

**Comment by KRline** [comment]
Plugin [External Media Without Import] is not the solution because you have to select all the images into WP media before starting to edit with Elementor. Apparently, Elementor only can see images that are already in the WP media at start of editing.
You can't add external media while editing the page with elementor. A real shame.

**Comment by yorkfrog** [comment]
@KRline - once in Elementor, if you add item to media library and want to access from Elementor, save page and then reload browser page (elementor) to pick up new media lib item.

**Comment by jaap115** [comment]
+1 for this option. It would be great to be able to add image sources. Would be great for building custom pages based on url's.

**Comment by Fogya** [comment]
@jaap115 Totally agree. Using image URLs and dynamic tags we could reach a whole new level of customisation.

**Comment by xueyongg** [comment]
it has been 4 years and nothing?!

**Comment by chichille** [comment]
If you don't want to use another plugin or if you have the PRO version of Elementor, the suite will not interest you.
Anyway the EAC plugin has an 'External image' Dynamic Tags which will allow you to add a link to an external image without using the media library.
The link can be created with the image widget and conserve all the properties and options that the widget provides.
In addition you can activate a fancybox to display an image or several images using the attributes of the links (Elementor v2.9.0)
Consult [this page to see the documentation](https://elementor-addon-components.com/add-external-image-for-elementor/)
One last thing, if you don't want to use the plugin's widgets, you can, from the plugin's configuration page, deactivate all the components and keep the Dynamic tags, Custom css and other features.
Rgds

**Comment by washedrepent** [comment]
Crazy its not added yet, considering how it could be made to work using the [media_sideload_image()](https://developer.wordpress.org/reference/functions/media_sideload_image/) function.

**Comment by eSilverStrike** [comment]
+1 for this. Want to create mockup of website first in Elementor and do not want to upload a bunch of files that are just placeholders and I will be deleting afterwards.

**Comment by lordloadalot** [comment]
ahahaha. no chance they will ever get this done. It's like begging tesla for a diesel option ...

**Comment by iweeleaks** [comment]
+1 here!!

**Comment by samheavyside** [comment]
> ahahaha. no chance they will ever get this done. It's like begging tesla for a diesel option ...

Not really. More like Tesla including GPS + Auto Drive in their cars but not allowing drivers to use the steering wheel to drive where they want.

**Comment by Ljuk67** [comment]
+1!!!!

**Comment by megaturba** [comment]
+1 please add this option!

**Comment by RBucci** [comment]
Holy sh!t.. They must really have something going on with not adding this option.. What the hell, people? I'm not updating my Pro License until this request is made.

**Comment by cirkut** [comment]
@KingYes @kobizz @matipojo @arielk Can we get SOME type of response please? Like seriously this is an actual problem that tons and tons of people have. I'm getting real sick and tired of Elementor simply adding new widgets and flashy stuff without addressing the real quality-of-life issues that people have.

Can we get a definitive yes or no on this?

**Comment by shilo-ey** [comment]
Hey all, we added this improvement to Elementor, you can try it out in the latest dev-edition, until officially released.

Thanks

**Comment by lordloadalot** [comment]
yess baby! ( gotta get back those clients wich escaped ... )

**Comment by cirkut** [comment]
@shilo-ey Thank you! We sincerely appreciate it! I apologize for the @mention spamming, but I feel like there are just some features that have slowly fallen by the wayside in lieu of other features (which are also great, but not as universally helpful to everyone).

I really appreciate the effort by the Elementor team and really hope the pricing changes will help push development faster, further, and better!

**Comment by cirkut** [comment]
Alright, here's a slightly updated request: After trying it out, it's great that we can paste a URL and it'll show by URL, but can we get this with Dynamic controls as well? Without that, we can't really make use of this in our templates, but in manual pages only.

I'm thinking about a template that pulls an image based on the post's ID for example (like https://example.com/images/756.jpg).

Overall this is definitely helpful, but if inserting from a URL, you should remove the image size selector (since it doesn't do anything) as well.

**Comment by shilo-ey** [comment]
This feature has been added to Elementor v3.3.0

Feel free to update

Thanks!

**Comment by lordloadalot** [comment]
jabbah dabbah doooooooh!
from "sorry, I din't understand the request" to feature in just five years. I  love you guys 🕺🏻.

**Comment by DaljitKhalsa** [comment]
Hi there, I am using Elementor and Cloudinary plugin and I have nearly 1825 images in my media library. Out of 1825, only 4 images got uploaded to Cloudinary and all others can not be synced as external images. Just created a support ticket about it. I recommend the plugin developers of Cloudinary should add this feature in the plugin where Cloudinary plugin should understand Elementor's overlay images + normal images. My website is [https://www.khalsa-website-designers.net/](https://www.khalsa-website-designers.net/) and on my website, there are more than 20 images per page/post.

**Comment by pwnoobz** [comment]
It's frustrating that I also couldn't find such an option when looking for it today. I don't know why this topic is closed.

**Comment by KRline** [comment]
hi pwnoobz, jou can just enter the URL to the cloudinary (or other) image when adding an image to Elementor now. Only disavantage with Cloudinary: you have to copy and paste the link manually between cloudinary website and elementor (and then change settings in the url for width and/or hight and quality... In fact you can use all cloudinary parameters in the URL such as  /f_auto,w_600/ for auto jpg/webp and width of 600 pixels)

**Comment by nevertoday** [comment]
I want use a custom metakey as thunmbnail, but elementor seem no this function.
Any tricks?

**Comment by Soetens** [comment]
> This feature has been added to Elementor v3.3.0
> 
> Feel free to update
> 
> Thanks!


What are the chances shortcodes will work?
